run.sh

```diff
@@ -48,6 +52,13 @@ prepare () {
   API_OUTPUT_US="${API_OUTPUT_DIR}/us"
 }
 
+execute_raw_data_qa() {
+  # Go to repo root (where run.sh lives).
+  RAW_DATA_OUTPUT_STREAM="/RAW_QA"
+  cd "$(dirname "$0")"
```

Comment:
Please replace the python lines
```
    if os.path.exists(args.output_dir):
        shutil.rmtree(args.output_dir)
```
with `rm "${API_OUTPUT_DIR}${RAW_DATA_OUTPUT_STREAM}"` here. I wouldn't expect `--output_dir` to be unconditionally cleared before new data is added. Expect your code to live much longer than you think/want so try to put as few surprises as possible in it.